pkg/account/component.go

```diff
@@ -541,3 +543,44 @@ func (c *component) executeActions(ctx context.Context, actions []string) error
 
 	return err
 }
+
+func (c *component) CancelEmailChange(ctx context.Context) error {
+	return c.removeStringAttributeFromUser(ctx, constants.AttrbEmailToValidate)
+}
+
+func (c *component) CancelPhoneNumberChange(ctx context.Context) error {
+	return c.removeStringAttributeFromUser(ctx, constants.AttrbPhoneNumberToValidate)
+}
+
+func (c *component) removeStringAttributeFromUser(ctx context.Context, attr kc.AttributeKey) error {
```

Comment:
No need to specify StringAttribute... We don't care if it contains a string or not.

Reply:
Are you sure about this? I'm using userKc.GetAttributeString below to check if the attribute is not present, so I guess I need to specify it

Reply:
All attributes are strings (more precisely, string arrays)

Reply:
Ok, then I'll change this